contributors/devel/contributor-cheatsheet.md

```diff
@@ -0,0 +1,39 @@
+# Kubernetes
+
+| Repo | PRs | Issues | Notes |
+| ---- | --- | ------ | ----- |
+| [k/kubernetes](https://github.com/kubernetes/kubernetes) | [PRs](https://github.com/kubernetes/kubernetes/pulls) | [Issues](https://github.com/kubernetes/kubernetes/issues) | [Meeting Notes](http://bit.ly/kubenotes)
+| [k/community](https://github.com/kubernetes/community) | [PRs](https://github.com/kubernetes/community/pulls) | [Issues](https://github.com/kubernetes/community/issues) |
+| [k/docs](https://github.com/kubernetes/kubernetes.github.io) | [PRs](https://github.com/kubernetes/kubernetes.github.io/pulls) | [Issues](https://github.com/kubernetes/kubernetes.github.io/issues)
+
+## Workflow
+
+- [Gubernator Dashboard - k8s.reviews](https://k8s-gubernator.appspot.com/pr)
+- [reviewable.kubernetes.io](https://reviewable.kubernetes.io/reviews#-)
+- [Submit Queue](https://submit-queue.k8s.io)
+- [Bot commands](https://github.com/kubernetes/test-infra/blob/master/commands.md)
+- [Release Buckets](http://gcsweb.k8s.io/gcs/kubernetes-release/)
+- [Developer Guide]
+  - [Cherry Picking Guide](https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md) - [Queue](http://cherrypick.k8s.io/#/queue)
+
+## SIGs and Working Groups
+
+- [SIG Master list](https://github.com/kubernetes/community/blob/master/sig-list.md)
```

Comment:
When I open the link, it says "Master SIG List" :-) - so may be good idea to use the same wording here (vs "SIG Master List") ? Also, the link is for this sub-topic so we should use this link - https://github.com/kubernetes/community/blob/master/sig-list.md#master-sig-list  And one more - since your providing sub-topics as link shouldn't we do both links - Master SIG List and "Master Working Group List" Thanks!